src/main/java/persistence/entity/EntityManager.java

```diff
@@ -0,0 +1,10 @@
+package persistence.entity;
+
+public interface EntityManager {
+
+    <T> T find(Class<T> clazz, Long Id);
+
+    Object persist(Object entity);
+
+    void remove(Object entity);
```

Comment:
```suggestion
public interface EntityManager<T> {

    T find(Class<T> clazz, Long Id);

    T persist(T entity);

    void remove(T entity);
```

미션 설명에는 Object 라고 나와있지만,
generic 을 적극적으로 활용해봐도 좋을 것 같습니다.